app/components/comment-item.js

```diff
@@ -62,13 +62,30 @@ export default Component.extend({
    * @property processedComment
    @ @type String
    */
-  processedBody: computed('comment.body', 'mentions', function() {
+  processedBody: computed('comment.body', 'comment.isLoaded', 'mentions', function() {
     let body = get(this, 'comment.body');
     let mentions = get(this, 'mentions');
-    return parse(body, mentions || []);
+    let parsedBody = parse(body, mentions || []);
+    return parsedBody;
```

Comment:
Also wanted to say I think we should just still write this as `return parse`. Is there a reason you rewrote this?

Reply:
Maybe we should rewrite `processedBody` to `parsedBody` generally? Any thoughts @begedin since I think you wrote this originally?

Reply:
Both seem to fit fine so I don't lean towards any of the names.

Reply:
I think that case then if we can change `processedBody` to `parsedBody` that'd be easier to grok.